frontend/tests/controllers-test/mainCtrl.test.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+describe('Unit tests for Main Controller', function () {
+    beforeEach(angular.mock.module('evalai'));
+
+    var $controller, createController, $rootScope, $scope, utilities, $state, vm;
+
+    beforeEach(inject(function (_$controller_, _$rootScope_, _$state_, _utilities_,) {
+        $controller = _$controller_;
+        $rootScope = _$rootScope_;
+        utilities = _utilities_;
+        $state = _$state_;
+
+        $scope = $rootScope.$new();
+        createController = function () {
+            return $controller('MainCtrl', {$scope: $scope});
+        };
+        vm = createController();
+    }));
+
+    describe('Global variables', function () {
+        it('has default values', function () {
+            vm = createController();
+            expect(vm.user).toEqual({});
+            expect(vm.challengeList).toEqual([]);
+            expect(vm.isChallenge).toBeTruthy({});
+            expect(vm.isMore).toBeFalsy();
```

Comment:
This `isMore` can be remove looks like it is cleaned up in main branch

Reply:
@galipremsagar done